website/docs/guides/v2-upgrade-guide.markdown

```diff
@@ -0,0 +1,154 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: Upgrade Guide for Kubernetes Provider v2.0.0"
+description: |-
+  This guide covers the changes introduced in v2.0.0 of the Kubernetes provider and what you may need to do to upgrade your configuration.
+---
+
+# Upgrading to v2.0.0 of the Kubernetes provider
+
+This guide covers the changes introduced in v2.0.0 of the Kubernetes provider and what you may need to do to upgrade your configuration.
+
+## Changes in v2.0.0
+
+### Changes to Kubernetes credentials supplied in the provider block
+
+We have made several changes to the way access to Kubernetes is configured in the provider block.
+
+1. The `load_config_file` attribute has been removed.
+2. Support for the `KUBECONFIG` environment variable has been dropped.
+3. The `config_path` attribute will no longer default to `~/.kube/config`.
+
+The above changes have been made to encourage the best practice of configuring access to Kubernetes in the provider block explicitly, instead of relying upon default paths or `KUBECONFIG` being set. We have done this because allowing the provider to configure its access to Kubernetes implicitly caused confusion with a subset of our users. It also created risk for users who use Terraform to manage multiple clusters. Requiring explicit configuring for kubernetes in the provider block eliminates the possibility that the configuration will be applied to the wrong cluster.
```

Comment:
```suggestion
The above changes have been made to encourage the best practice of configuring access to Kubernetes in the provider block explicitly, instead of relying upon default paths or `KUBECONFIG` being set. We have done this because allowing the provider to configure its access to Kubernetes implicitly caused confusion with a subset of our users. It also created risk for users who use Terraform to manage multiple clusters. Requiring explicit configuration for Kubernetes in the provider block eliminates the possibility that the configuration will be applied to the wrong cluster.
```